README.md

```diff
@@ -16,18 +16,84 @@ The Keptn Lifecycle Controller is composed of the following components:
 The Keptn Lifecycle Operator contains several controllers for Keptn CRDs and a Mutating Webhook.
 The Keptn Scheduler ensures that Pods are started only after the pre-deployment checks have finished.
 
-## Architecture
 
-![](./assets/architecture.png)
+## Goals
+
+The Keptn Lifecycle Controller aims at supporting Cloud Native teams with:
+
+- Pre-requisite evaluation before deploying Workloads and applications
+- Finding out when an application (not workload) is ready and working
+- Checking the Application Health in a declarative (cloud-native) way
+- Standardized way for pre- and post-deployment tasks
+- Provide out-of-the-box Observability of the deployment cycle
+
+![](./assets/operator-maturity.jpg)
+
+The Keptn Lifecycle Controller could be seen as a general purpose and declarative Level 3 operator for your Application.
+For this reason, the Keptn Lifecycle Controller is agnostic to the Deployment tool that is being used and works with any GitOps solution.
+
+## How to install
+
+**Prerequisites:**
+
+The lifecycle controllers includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod. The certificate creation
+is handled automatically by [cert-manager](https://cert-manager.io). To install **cert-manager**, follow their [installation instructions](https://cert-manager.io/docs/installation/).
+
+
+When the *cert-manager* is installed, you can run
+
+```bash
+kubectl apply -f <insert link of release>
+```
+
+## How to use
+
+The Keptn Lifecycle Controller monitors manifests that are applied against the KubeAPI and reacts if it finds a workload with special annotations.
```

Comment:
That's not entirely correct, as manifests are already applied when pods are changed by the mutating webhook ...